layouts/chapter.html

```diff
@@ -61,5 +61,15 @@
   </div>
 </div>
 
+<script>
```

Comment:
The folks at Google delineated this section with a [comment](https://developers.google.com/analytics/devguides/collection/analyticsjs/) describing the snippet.

Probably not necessary, but it *is* a pretty ugly snippet that could use some sort of license/identification. :oncoming_police_car: